shared/src/validation-schemas/sign-up/sign-up.schema.ts

```diff
@@ -0,0 +1,15 @@
+import * as Joi from 'joi';
+import { UserCreatePayload } from '~/common/types/types';
+
+const SignupSchema = Joi.object<UserCreatePayload>({
```

Comment:
```suggestion
const signUp = Joi.object<UserCreatePayload>({
```
Check the **A3 criteria**